js/views/modals/listingDetail/Listing.js

```diff
@@ -426,6 +426,11 @@ export default class extends BaseModal {
       vendor: this.vendor,
       removeOnClose: true,
       showCloseButton: false,
+      initialState: {
+        isFetching: true,
+        fetchError: '',
+        fetchFailed: false,
```

Comment:
In the future it would be good to move more of the template to using the state, I spent some time retrofitting it, but backed those changes out because it would require a lot of refactoring. For now the state only handles the initial load and any errors.